kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/coroutines/provokeThreadSwitch.kt

```diff
@@ -0,0 +1,10 @@
+package com.sksamuel.kotest.engine.coroutines
+
+/**
+ * Provoke the dispatcher to switch coroutine threads by keeping the coroutine alive for a sufficient time period.
+ */
+@Suppress("RedundantSuspendModifier")
+suspend fun provokeThreadSwitch() {
```

Comment:
How come this is suspend ?

Reply:
Sure, technically it does not. The idea is to communicate that this function is meant to be invoked only in a coroutine context. So more like a poor man's code coloring. Would you favor a change or added comment?

Reply:
I guess if we run it outside of suspend it doesn't matter, maybe make it internal, and add a comment ?

Reply:
Like so?

```kotlin
/**
 * Provoke the dispatcher to switch coroutine threads by keeping the coroutine alive for a sufficient time period.
 *
 * This function is not suspending, but intended to be invoked in a coroutine exclusively.
 */
internal fun provokeThreadSwitch() {
   Thread.sleep(50)
}
```

Reply:
I like it

Reply:
Done via 63c927a9a4e52bb193a865e8995707d865cb63ae